client/src/Features/MicrosoftLearn/MicrosoftLearnSearch.tsx

```diff
@@ -20,6 +20,7 @@ const MicrosoftLearnSearchInner = ({ styles }: IStylesOnly<MicrosoftLearnSearchS
           onChange={(_e, newValue) => learnStore.updateSearchTerm(newValue || '')}
           className={classes.searchBox}
           disabled={!!learnStore.isLoadingCatalog}
+          value={learnStore.searchTerm}

```

Comment:
Yes, the `value` property addition. I added this to facilitate persistence on page-reloads. So whenever the user refreshes the page, the store updates the `learnstore.searchTerm` from query parameters, and that gets reflected here as well. It was missing earlier probably because it wasn't needed, but this seemed the way to reflect value changes easily to the UI `SearchBox`. Should we be doing it some other way?

Reply:
No, this looks fine.